licensing/src/main/java/io/crate/license/LicenseService.java

```diff
@@ -117,13 +131,13 @@ private void registerSelfGeneratedLicense(ClusterState clusterState) {
         DiscoveryNodes nodes = clusterState.getNodes();
         if (nodes != null) {
             if (nodes.isLocalNodeElectedMaster()) {
-                DecryptedLicenseData licenseData = new DecryptedLicenseData(Long.MAX_VALUE, clusterState.getClusterName().value());
+                DecryptedLicenseData licenseData = new DecryptedLicenseData(
+                    System.currentTimeMillis() + 22 * 3600 * 1000, clusterState.getClusterName().value());
```

Comment:
```suggestion
                    System.currentTimeMillis() + 30L * 24 * 3600 * 1000, clusterState.getClusterName().value());
```

Reply:
That was a forgotten test change. Dropped it completely